nrf_security/Kconfig

```diff
@@ -1524,6 +1524,18 @@ config MBEDTLS_SSL_CIPHERSUITES
 	  Warning: This field has offers no validation checks.
 	  MBEDTLS_SSL_CIPHERSUITES setting in mbed TLS config file.
 
+config MBEDTLS_PK_C
```

Comment:
I see that `MBEDTLS_PK_C` is automatically enabled in the mbedtls header when `CONFIG_MBEDTLS_X509_LIBRARY=y`
https://github.com/nrfconnect/sdk-nrfxlib/blob/master/nrf_security/cmake/kconfig_mbedtls_configure.cmake#L270

so wonder if `MBEDTLS_X509_LIBRARY` config should have a `select MBEDTLS_PK_C` to make this visible to the user.

Reply:
I have added the `select` statement to `MBEDTLS_X509_LIBRARY`.